package.json

```diff
@@ -33,7 +33,8 @@
     "chai": "^4.2.0",
     "flow-bin": "^0.70.0",
     "grumbler-scripts": "^3",
-    "mocha": "^4"
+    "mocha": "^4",
+    "sinon": "^7.3.2"
```

Comment:
should sinon be removed?

Reply:
I just updated the description to include the error I'm getting when it's not removed. I get this error `ERROR [karma-server]: Server start failed on port 9876: Error: Cannot find module 'sinon/package.json'`